src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <fullName>gseuChecklistItemCaptureDonorIntentTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>Capture Donor Intent with Donation Allocations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Monthly giving can provide a predictable source of nonprofit revenue, and sustaining donors are often a nonprofit&apos;s most loyal and committed supporters. Track monthly giving by creating a Recurring Donation.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Learn more about Recurring Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Create Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Collect Sustaining Gifts with Recurring Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Individual in GS for End User</shortDescription>
+        <value>For most nonprofits, donations come from individual contributions. Track individual donations by creating an Opportunity from a Contact record.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationIndividualLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Donation Individual in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Link label of Individual Donation checklist item in Get Started for End Users.